client/network/light/src/light_client_requests/handler.rs

```diff
@@ -54,7 +54,12 @@ pub struct LightClientRequestHandler<B, Client> {
 impl<B, Client> LightClientRequestHandler<B, Client>
 where
 	B: Block,
-	Client: BlockBackend<B> + ProofProvider<B> + Send + Sync + 'static,
+	Client: BlockBackend<B>
+		+ ProofProvider<B>
+		+ BlockIdTo<B, Error = sp_blockchain::Error>
```

Comment:
Is this `BlockIdTo` bound actually required here?

Reply:
Nope (7083fa3)